Gemfile

```diff
@@ -83,5 +83,7 @@ gem "kaminari"
 gem "paper_trail"
 gem "phonelib"
 gem "sendgrid-ruby"
+gem "simple_form"
+gem "simple_form-tailwind"
```

Comment:
Så vidt jeg kan forstå så legger denne til noen standard stylings for forms, som bestemmes i en config et sted? 

Vil ikke det bety at vi har da tre ulike steder hvor CSS kan bestemmes: Enten i SCSS-flier, eller i HTML-en med classes, eller i config-filene til simple forms? For meg så virker det litt mye - men kan være jeg tar feil!

Eller, hvordan fungerer simple_forms-tailwind - og hvorfor er den kul?

Reply:
Simple_forms gir oss den der error_handlingen gratis. 

Vi bruker tailwind og den letteste måten jeg fant for å style alle simple_form "tingene" på var den gem'en. 
Den setter opp en config fil hvor vi enkelt kan konfigurere alle parametre angående simple forms, sette opp defaults, og slikt. Da slipper vi å legge til samme klasser på hvert skjema og input.